### PR TITLE
Translation Engine for Gondi and APIs for Translation Engine

### PR DESCRIPTION
**Pull request by anuragshukla06**
Hi! I have added the rest APIs for translation queries and also Gondi language support for both in web and API

### CODE REVIEW

**Comment by anuragshukla06** [comment]
> I haven't yet gone into the details of the API implementation, however, want to point to what I think we want by an API setup.
> 
> With the current implementation - there are 3*3=9 translation engines running - not efficient. What we ideally want is to decouple in such a way that there are only 2 translation engines running on an API, which gets queried by the independent views.py, be it mt/mtsimple.

Okay, I will do t

File: mt/migrations/0015_auto_20200519_1612.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Generated by Django 3.0.3 on 2020-05-19 16:12
```
Comment (SebastinSanty) [inline mt/migrations/0015_auto_20200519_1612.py:1]: Remove this file

File: docs/openapi-yaml-client-generated/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+openapi: 3.0.0
```
Comment (SebastinSanty) [inline docs/openapi-yaml-client-generated/openapi.yaml:1]: Why are these documentations getting generated?

Comment (anuragshukla06) [reply docs/openapi-yaml-client-generated/openapi.yaml:1]: These documentation are for the APIs

File: mt/migrations/0015_auto_20200519_1612.py (hunk 1 above)
Comment (anuragshukla06) [reply mt/migrations/0015_auto_20200519_1612.py]: Done

**Comment by SebastinSanty** [comment]
I have pushed the squashed commits directly to master branch.

**Pull request closed without merging**